utilities/load_testing/README.md

```diff
@@ -39,3 +49,10 @@ To run API load tests against a local API instance, use `just api`. You can
 optionally specify a host (the default is to point to local). For example
 `just api https://api-staging.openverse.org` will run the load tests against the
 staging API.
+
+To run the frontend load tests, use `just k6-frontend`. You can optionally
+specify the scenarios to run, for example, `just k6-frontend all`. To specify a
+host (the default is to point to the `openverse.org`), set the `FRONTEND_URL`
+environment variable. For example,
+`FRONTEND_URL=https://nuxt-preview.openverse.org just k6-frontend` will run the
+load tests against the production frontend.
```

Comment:
```suggestion
environment variable. For example,
`FRONTEND_URL=https://nuxt-preview.openverse.org just k6-frontend` will run the
load tests against the production frontend.
```
```suggestion
environment variable. For example,
`FRONTEND_URL=https://staging.openverse.org just k6-frontend` will run the
load tests against the staging frontend.
```

Staging might be the best example, since we're using production for the default, and I think you just forgot to update the domain.